eslint.config.js

```diff
@@ -17,14 +17,31 @@ export default defineConfig([
   globalIgnores(['dist']),
   {
     extends: [
+      // https://github.com/eslint/eslint/blob/20a9e59438fde3642ab058cc55ee1b9fa02b6391/packages/js/src/configs/eslint-recommended.js
```

Comment:
It's not easy to figure out the total effective ruleset. But these comments should help. I added them when I added [this rule](https://github.com/CrowdStrike/glide-core/pull/910/files#diff-a32a0887ed9d1d707bbb3b845b7df7fd40e673c47e7b60a3ebd896b68d3b8839R98)

TODO